InjectionBundle/SwiftEval.swift

```diff
@@ -220,7 +220,7 @@ public class SwiftEval: NSObject {
                         if ($line =~ /^\\s*cd /) {
                             $realPath = $line;
                         }
-                        elsif (my ($product) = $line =~ m@\\.xcent --timestamp=none (.*)\\r@o) {
+                        elsif (my ($product) = $line =~ m@\\.xcent --timestamp=none (.*\\.app)\\r@o) {
```

Comment:
Curious to why this shows up, this should already be on `master`

Reply:
It’ll all come out in the wash but as you merge can you be sure to Squash?